onnxruntime/contrib_ops/cpu/transformers/subgraph_t5_decoder.cc

```diff
@@ -127,28 +128,32 @@ Status T5DecoderSubgraph::CreateInitialFeeds(
     const std::vector<OrtValue>& encoder_feeds,
     const std::vector<OrtValue>& encoder_fetches,
     std::vector<OrtValue>& decoder_feeds,
-    const GenerationDeviceHelper::DeviceCopyFunc<int32_t>& device_copy_int32_func,
     const GenerationDeviceHelper::ExpandBufferFunc<int32_t>& expand_buffer_int32_func,
     const GenerationDeviceHelper::ExpandBufferFunc<float>& expand_buffer_float_func,
     const GenerationDeviceHelper::ExpandBufferFunc<MLFloat16>& expand_buffer_float16_func,
     int num_beam,
-    void* stream) {
+    void* stream,
+    int cur_len,
+    transformers::Sequences& sequences) {
   ORT_ENFORCE(session_state_ != nullptr, "Setup must be called before CreateInitialFeeds");
 
   // Allocate subgraph inputs from same device as inputs of encoder subgraph.
   AllocatorPtr allocator = session_state_->GetAllocator(encoder_feeds[0].Get<Tensor>().Location());
 
   // Copy beam next tokens in CPU to input_ids in provider device (CPU for CPU EP, or GPU for CUDA EP).
   int batch_beam_size = static_cast<int>(beam_next_tokens.size());
-  int64_t dims[] = {batch_beam_size, 1};
```

Comment:
please make this feature as optional otherwise it will fail the general case. ref: https://github.com/microsoft/onnxruntime/blob/main/onnxruntime/contrib_ops/cpu/transformers/subgraph_t5_decoder.cc#L84 when this var is true then use sequence as next input_ids